game_stats_fixtures/game_teams_fixtures.csv

```diff
@@ -0,0 +1,39 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```

Comment:
@ajtran303 I went through and added all of your game_teams_fixtures from your branch so it should match your file exactly